docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -1,23 +1,155 @@
 Semantic Segmentation
 =====================
 
+Semantic segmentation is a computer vision task in which an algorithm assigns a label or class to each pixel in an image.
+For example, semantic segmentation can be used to identify the boundaries of different objects in an image, such as cars, buildings, and trees.
+The output of semantic segmentation is typically an image where each pixel is colored with a different color or label depending on its class.
+
+.. _semantic_segmentation_image_example:
+
+.. image:: ../../../../utils/images/semantic_seg_example.png
+  :width: 600
+  :alt: image uploaded from this `source <https://arxiv.org/abs/1912.03183>`_
+
+
+We solve this task by utilizing `FCN Head <https://arxiv.org/pdf/1411.4038.pdf>`_ with implementation from `MMSegmentation <https://mmsegmentation.readthedocs.io/en/latest/_modules/mmseg/models/decode_heads/fcn_head.html>`_ on the multi-level image features obtained by the feature extractor backbone (`Lite-HRNet <https://arxiv.org/abs/2104.06403>`_).
+For the supervised training we use the following algorithms components:
+
+.. _semantic_segmentation_supervised_pipeline:
+
+- ``Augmentations``: Besides basic augmentations like random flip, random rotate and random crop, we use mixing images technique with different `photometric distortions <https://mmsegmentation.readthedocs.io/en/latest/api.html#mmseg.datasets.pipelines.PhotoMetricDistortion>`_.
+
+- ``Optimizer``: We use `Adam <https://arxiv.org/abs/1412.6980>`_ optimizer with weight decay set to zero and gradient clipping with maximum quadratic norm equals to 40.
+
+- ``Learning rate schedule``: For scheduling training process we use **ReduceLROnPlataeu** with linear learning rate warmup for 100 iterations. This method monitors a target metric (in our case we use metric on the validation set) and if no improvement is seen for a 'patience' number of epochs, the learning rate is reduced.
+
+- ``Loss function``: We use standart `Cross Entropy Loss <https://en.wikipedia.org/wiki/Cross_entropy>`_  to train a model.
```

Comment:
@JihwanEom Do you think we need to explain the ignore management in loss?

Reply:
Why we silently substitute losses and other components in config?
![image](https://user-images.githubusercontent.com/68103869/216462093-8edd066e-e1f4-4b45-8813-0fc3d863177a.png)
task_adapt always True for each config-> why can't we directly point which loss we are using in config? Not just `Cross Entropy`, but `CrossEntropyWithIgnore`

Reply:
Anyway, it is for class incremental -> for incremental learning exists an additional section below which hasn't been filled yet

Reply:
Yeah if we have a additional section for incremental, it would be good to explain in there.